hscontrol/types/node.go

```diff
@@ -397,12 +386,15 @@ func (node *Node) Proto() *v1.Node {
 		GivenName:   node.GivenName,
 		User:        node.User.Proto(),
 		ForcedTags:  node.ForcedTags,
-		HostInfo:    node.HostInfoAsProtoStruct(),
 
 		// TODO(kradalby): Implement register method enum converter
 		// RegisterMethod: ,
 
 		CreatedAt: timestamppb.New(node.CreatedAt),
+
+		TailCfg: &v1.TailCfg{
```

Comment:
This intermediate type seem unnecessary, just put HostInfo straight in the Node object.